web/src/lib/client/manager.js

```diff
@@ -87,6 +88,16 @@ export default class ManagerClient {
       handler(data, invalid);
     });
   }
+
+  /**
+   * Returns whether calling the system reboot suceeded or not.
+   *
+   * @return {Promise.<boolean>}
+   */
+  rebootSystem() {
+    console.log("Rebooting the system");
```

Comment:
NP: I guess this console output is not needed, although it does not hurt.

Reply:
I will remove it.